### PR TITLE
10 authors per page, link to user profile

### PR DESCRIPTION
**Pull request by Docker-J**

### CODE REVIEW

File: src/pages/Friends/followed.js [hunk 1]
```diff
@@ -48,109 +46,108 @@ function Followed() {
...
-    } 
-    else if (follow_list.items.length < 5)
-    {
-      return <button onClick={back_page}>Prev Page</button>
-    } 
-    else 
-    {
+    if (page === 1) {
+      return <button onClick={forward_page}>Next Page</button>;
+    } else if (follow_list.items.length < 5) {
```
Comment (laingkob) [inline src/pages/Friends/followed.js:57]: Should this be 10 now?

File: src/pages/Friends/realfriends.js [hunk 2]
```diff
@@ -49,104 +49,105 @@ function Realfriends() {
 
   let page = 1;
   const page_buttons = () => {
-
-    if (follow_list.items.length < 5 && page === 1)
-    {
+    if (follow_list.items.length < 5 && page === 1) {
```
Comment (laingkob) [inline src/pages/Friends/realfriends.js:52]: Also here?

File: src/pages/Friends/realfriends.js [hunk 3]
```diff
@@ -49,104 +49,105 @@ function Realfriends() {
...
-    } 
-    else if (follow_list.items.length < 5)
-    {
-      return <button onClick={back_page}>Prev Page</button>
-    } 
-    else 
-    {
+    if (page === 1) {
+      return <button onClick={forward_page}>Next Page</button>;
+    } else if (follow_list.items.length < 5) {
```
Comment (laingkob) [inline src/pages/Friends/realfriends.js:57]: And here?

File: src/pages/Friends/request.js [hunk 4]
```diff
@@ -20,163 +20,149 @@ import TableHead from "@mui/material/TableHead";
...
-      navigate("/");
+    navigate("/");
   };
 
   let page = 1;
   const page_buttons = () => {
-
-    if (follow_list.items.length < 5 && page === 1)
-    {
+    if (follow_list.items.length < 5 && page === 1) {
```
Comment (laingkob) [inline src/pages/Friends/request.js:59]: And here

File: src/pages/Friends/request.js [hunk 5]
```diff
@@ -20,163 +20,149 @@ import TableHead from "@mui/material/TableHead";
...
-    } 
-    else if (follow_list.items.length < 5)
-    {
-      return <button onClick={back_page}>Prev Page</button>
-    } 
-    else 
-    {
+    if (page === 1) {
+      return <button onClick={forward_page}>Next Page</button>;
+    } else if (follow_list.items.length < 5) {
```
Comment (laingkob) [inline src/pages/Friends/request.js:64]: And here

Comment (Docker-J) [reply src/pages/Friends/request.js:64]: I just updated get_all_authors api to get 10 authors so leave other as is for now but yeah I can do that for other calls

**Review by laingkob — APPROVED** [review]
Thanks for working on this so quickly. I think a couple of value changes were missed- I marked the ones I saw but there might be a couple more in other files. This way we will be showing page buttons inaccurately, but I dont think that effects the overall usability

**Pull request merged**